#===================================================================================================
# Source: https://github.com/FFmpeg/FFmpeg/commit/e5163b1d34381a3319214a902ef1df923dd2eeba.patch
#---------------------------------------------------------------------------------------------------
# From: Christopher Degawa
# Date: Wed, 11 May 2022 15:11:04 -0500
# Subject: [PATCH] configure: extend SDL check to accept all 2.x versions
#
# sdl2 recently changed their versioning, moving the patch level to minor level
# https://github.com/libsdl-org/SDL/commit/cd7c2f1de7d9e418bb554047d714dd7cacc020ff
# and have said that they will instead ship sdl3.pc for 3.0.0
#
# Fixes ticket 9768
#===================================================================================================
--- configure.orig
+++ configure
@@ -6742,7 +6742,7 @@ fi
 
 if enabled sdl2; then
     SDL2_CONFIG="${cross_prefix}sdl2-config"
-    test_pkg_config sdl2 "sdl2 >= 2.0.1 sdl2 < 2.1.0" SDL_events.h SDL_PollEvent
+    test_pkg_config sdl2 "sdl2 >= 2.0.1 sdl2 < 3.0.0" SDL_events.h SDL_PollEvent
     if disabled sdl2 && "${SDL2_CONFIG}" --version > /dev/null 2>&1; then
         sdl2_cflags=$("${SDL2_CONFIG}" --cflags)
         sdl2_extralibs=$("${SDL2_CONFIG}" --libs)
